sinks/hawkular/driver.go

```diff
@@ -78,7 +106,11 @@ func (self *hawkularSink) Register(mds []sink_api.MetricDescriptor) error {
 
 // Fetches definitions from the server and checks that they're matching the descriptors
 func (self *hawkularSink) updateDefinitions(mt metrics.MetricType) error {
-	mds, err := self.client.Definitions(metrics.Filters(metrics.TypeFilter(mt)))
+	m := make([]metrics.Modifier, len(self.modifiers), len(self.modifiers)+1)
```

Comment:
Not that it'll make too much difference given the probable size of the slice, but do you actually want to copy the underlying data?